BlueprintUICommonControls/Sources/AccessibilityElement.swift

```diff
@@ -131,6 +137,42 @@ public struct AccessibilityElement: Element {
     }
 }
 
+extension AccessibilityElement.Trait: Hashable, Equatable {
+
+    public var rawValue: Int {
+        switch self {
+        case .button: return 0
+        case .link: return 1
+        case .header: return 2
+        case .searchField: return 3
+        case .image: return 4
+        case .selected: return 5
+        case .playsSound: return 6
+        case .keyboardKey: return 7
+        case .staticText: return 8
+        case .summaryElement: return 9
+        case .notEnabled: return 10
+        case .updatesFrequently: return 11
+        case .startsMediaSession: return 12
+        case .adjustable: return 13
+        case .allowsDirectInteraction: return 14
+        case .causesPageTurn: return 15
+        case .tabBar: return 16
+        }
+    }
+
+    public static func == (lhs: AccessibilityElement.Trait, rhs: AccessibilityElement.Trait) -> Bool {
+        lhs.rawValue == rhs.rawValue
+    }
+
+    public var hashValue: Int {
+        rawValue
+    }
```

Comment:
Pretty sure this is deprecated and you only need to add `hash(into:)` for `Hashable` conformance

Reply:
removed